pkg/api/defaults-controller-manager.go

```diff
@@ -60,6 +61,9 @@ func (cs *ContainerService) setControllerManagerConfig() {
 	// Enables Node Exclusion from Services (toggled on agent nodes by the alpha.service-controller.kubernetes.io/exclude-balancer label).
 	addDefaultFeatureGates(o.KubernetesConfig.ControllerManagerConfig, o.OrchestratorVersion, "1.9.0", "ServiceNodeExclusion=true")
 
+	// Enable the consumption of local ephemeral storage and also the sizeLimit property of an emptyDir volume.
```

Comment:
can the tests be updated for this?

Reply:
my bad...i added a test for all default feature-gates since it was missing.